app/javascript/packs/application.js

```diff
@@ -1,7 +1,27 @@
+import 'babel-polyfill'
```

Comment:
I've moved it here from `App/index.js` because it's not related to app business logic.